ddtrace/tracer/rules_sampler.go

```diff
@@ -565,24 +606,28 @@ func unmarshalSamplingRules(b []byte, spanType SamplingRuleType) ([]SamplingRule
 				errs = append(errs, fmt.Sprintf("at index %d: rate not provided", i))
 				continue
 			}
-			rate, err := v.Rate.Float64()
-			if err != nil {
-				errs = append(errs, fmt.Sprintf("at index %d: %v", i, err))
-				continue
-			}
-			if rate < 0.0 || rate > 1.0 {
-				errs = append(errs, fmt.Sprintf("at index %d: ignoring rule %+v: rate is out of [0.0, 1.0] range", i, v))
-				continue
-			}
-
 			switch {
 			case v.Service != "" && v.Name != "":
 				rules = append(rules, NameServiceRule(v.Name, v.Service, rate))
+				continue
 			case v.Service != "":
 				rules = append(rules, ServiceRule(v.Service, rate))
+				continue
 			case v.Name != "":
 				rules = append(rules, NameRule(v.Name, rate))
+				continue
+			}
+			var targetRoot bool
+			switch v.TargetRoot {
```

Comment:
Wouldn't hurt to be case-insensitive here
```suggestion
			switch strings.ToLower(v.TargetSpan) {
```